tests/test_download.py

```diff
@@ -168,44 +170,29 @@ def test_download_url_to_tempfileobj_and_urls(self):
     self.assertRaises(securesystemslib.exceptions.FormatError,
                       download_file, None, self.target_data_length)
 
-    self.assertRaises(securesystemslib.exceptions.FormatError,
+    self.assertRaises(requests.exceptions.MissingSchema,
```

Comment:
The code no longer raises MissingSchema, since it now parses URLs to extract the hostname. It instead raises securesystemslib.exceptions.FormatError, so we'd have to test for that instead. That said, it's probably best to use a different error type, since while this is, true enough, a formatting error, FormatError is customarily reserved for the automatic detection based on schemas in formats.py (using `<SCHEMA>.check_match()`)